lib/modules/dosomething/dosomething_rogue/dosomething_rogue.install

```diff
@@ -252,3 +252,40 @@ function dosomething_rogue_update_7006(&$sandbox) {
     }
   }
 }
+
+/**
+ * Adds source, remote_addr, failed_at, and tries columns to dosomething_rogue_failed_task_log.
+ */
+function dosomething_rogue_update_7007(&$sandbox) {
```

Comment:
I think you should be able to add these fields in a loop. Like use an array with each field's settings and then loop over the array to execute the repeated parts.